app/controllers/support_interface/course_controller.rb

```diff
@@ -10,6 +10,7 @@ def applications
 
     def vacancies
       @course = Course.find(params[:course_id])
+      @course_options = CourseOption.where(course: @course).includes(:site, :course)
```

Comment:
Can we open a separate PR for the commit including this fix, as it's unrelated to the API changes.

Reply:
Normally I'd agree! But in this case the spec failure associated with this emerged following other changes in this PR, so we'll end up with a mixed PR anyway